.github/workflows/build_and_deploy.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   build-and-deploy:
     strategy:
       matrix:
-        python3-minor-version: [7, 8, 9, 10, 11, 12]
+        python3-minor-version: [9, 10, 11]
```

Comment:
Think this was a bad conflict resolution. We support 3.12 today

Reply:
Good catch